go/auth/credential_authority.go

```diff
@@ -33,6 +33,8 @@ type checkArg struct {
 	uid      keybase1.UID
 	username *libkb.NormalizedUsername
 	kid      *keybase1.KID
+	sibkeys  *[]keybase1.KID
+	subkeys  *[]keybase1.KID
```

Comment:
Is it necessary to have `*[]` rather than just `[]` and signal "not-interested" via `nil` slice, and "empty set" via 0-length slice?